app/code/Magento/Ui/TemplateEngine/Xhtml/Result.php

```diff
@@ -56,21 +61,24 @@ class Result implements ResultInterface
      * @param Structure $structure
      * @param LoggerInterface $logger
      * @param JsonHexTag $jsonSerializer
+     * @param State|null $state
      */
     public function __construct(
         Template $template,
         CompilerInterface $compiler,
         UiComponentInterface $component,
         Structure $structure,
         LoggerInterface $logger,
-        JsonHexTag $jsonSerializer = null
+        JsonHexTag $jsonSerializer = null,
```

Comment:
Replace JsonHexTag by ?JsonHextTag since it can be nullable.

Reply:
done